### PR TITLE
Create .gitignore

### PR DESCRIPTION
**Pull request by zaynaib**
Created a gitignore file. Are there any paths or files your team want to be ignored?

### CODE REVIEW

**Comment by jameslamb** [comment]
Hey @zaynaib thanks for doing this!

If you're not sure where to start, I think this is a good set for an R project:

```
# R stuff
*.Rhistory
*.Rproj
*.rda
*.rds
*.Rdata

# data files
*.csv
**/*.csv
**/*.xlsx
**/*.pdf

# other misc stuff
*.DS_Store
```

**Review by jameslamb — CHANGES_REQUESTED** [review]
Can you please add the defaults I suggested? Then we will merge this up!

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+# This a file which Git has been explicitly told to ignore
```
Comment (jameslamb) [inline .gitignore]: can you change this text? It's not actually that Git ignores _this file_.

Recommendation:

```
# This is a file telling Git what other files to ignore
```

Comment (zaynaib) [reply .gitignore]: Got it. Sorry for the grammar issues.

**Comment by jameslamb** [comment]
Great thanks! Don't worry about the failing Travis stuff...totally unrelated to your code.

@cwschultz88 from my perspective, this PR is good to go.

@zaynaib We really appreciate you making this contribution and hope you'll come back to make more!

**Review by jameslamb — APPROVED** [review]

**Comment by zaynaib** [comment]
@jameslamb Two questions. 1.) Do you know of any resources where I can about vignette with R?
2.) Is there anything else you need me to do for this PR? Or any other issues you need help with?

**Comment by jameslamb** [comment]
@zaynaib Nothing else needed for this particular PR! @cwschultz88 is the maintainer so he'll be the one to merge this.

If you're interested in learning about vignettes in R packages, Hadley's book is a great resource: http://r-pkgs.had.co.nz/vignettes.html

If you're looking for other issues to pick up, check out [pkgnet!](https://github.com/UptakeOpenSource/pkgnet/issues). We have quite a few open issues, and perhaps looking through the code you'll find more. Any contributions are welcomed!

**Comment by cwschultz88** [comment]
Everything looks great -- thanks for the contribution!

**Pull request merged**